README.md

```diff
@@ -96,9 +86,6 @@ Import the required files and include mixins in the right order so consuming app
 
 ```
 ember install @fortawesome/ember-fontawesome
```

Comment:
You don't have to manually install the icon sets, but you do still have to install the fontawesome addon?

Reply:
I was having trouble getting my local dev using `yarn link` to get `ember-truth-helpers` to work in the project consuming `labs-ui` without running `ember install ember-truth-helpers` -- I wasn't sure why but it was taking too long so for the sake of caution and time I kept the steps that use `ember install` -- not entirely sure if there's additional steps that yarn dependency resolution is _not_ covering.  It's totally possible that this could be unnecessary and can investigate at another time!